src/game/sdl2/touch.inc

```diff
@@ -180,6 +180,29 @@ begin
   Result := found and (xx >= x) and (yy >= y) and (xx <= x + w) and (yy <= y + h);
 end;
 
+function TouchIsWanted (): Boolean; inline;
+begin
+{$IFDEF ANDROID}
+  // If we are on Android, always enable touchscreen in menu, so that player doesn't get locked out.
```

Comment:
Why this comment moved above? It was more reasonable near the specific condition.